Sources/VGSCollectSDK/Core/Enums.swift

```diff
@@ -96,8 +96,7 @@ internal extension FieldType {
         case .cardHolderName:
             return "^([a-zA-Z0-9\\ \\,\\.\\-\\']{2,})$"
         case .ssn:
-            return
-          "^(?!\\b(\\d)\\1+\\b)(?!(123456789|219099999|078051120|457555462))(?!(000|666|9))(\\d{3}-?(?!(00))\\d{2}-?(?!(0000))\\d{4})$"
```

Comment:
123456789 is not valid SSN, why it's removed?

Reply:
Hi @dmytrokhl, it is updated because we need to use the same regex in all the platforms, following the patterns described here: https://verygoodsecurity.atlassian.net/wiki/spaces/COLLECT/pages/99156598/Comparing+Collect+CC+RegExp